scss/_variables.scss

```
@@ -882,29 +882,29 @@ $table-cell-icon-margin-bottom:         -.625rem !default; // Boosted mod
 $table-cell-vertical-align:             top !default;
 $table-line-height:                     1.25 !default; // Boosted mod
 
-$table-color:                           var(--#{$prefix}body-color) !default;
+$table-color:                           var(--#{$prefix}emphasis-color) !default; // Boosted mod: instead of `var(--#{$prefix}emphasis-color)`
```

```suggestion
$table-color:                           var(--#{$prefix}emphasis-color) !default;
```

or

```suggestion
$table-color:                           var(--#{$prefix}body-color) !default; // Boosted mod: instead of `var(--#{$prefix}emphasis-color)`
```

It's not exactly the same in dark mode compared to the main branch, but I'm fine with this since it'll be tackled by our dark mode.